eslint.config.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
+
+const js = require("@eslint/js");
+const { FlatCompat } = require("@eslint/eslintrc");
+const eslintConfigESLintCJS = require("eslint-config-eslint/cjs");
+const globals = require("globals");
+const reactPlugin = require("eslint-plugin-react");
+const reactRecommended = require("eslint-plugin-react/configs/recommended");
+const jsxA11yPlugin = require("eslint-plugin-jsx-a11y");
+const reactHooksPlugin = require("eslint-plugin-react-hooks");
+
+const compat = new FlatCompat({
+    baseDirectory: __dirname
```

Comment:
do we need this? doesn't this resolve to the current working directory?

Reply:
We don't need FlatCompat anymore